### ISSUE TITLE
rancher2_cluster fails to provision against rancher 2.6.0

### ISSUE THREAD

**Issue by mgherman** [rancher/terraform-provider-rancher2#742]
Hi Folks,

Im unsure of whether this should be currently being supported, but ill ask anyway. When attempting to provision a new rancher2_cluster against a 2.6.0 Rancher, using an RKE template, the provisioning is successful, however terraform reports a failure due to the `generateKubeconfig` call returning a 503.

```
rancher2_cluster.tf-test: Refreshing state... [id=c-lrxnb]
╷
│ Error: Bad response statusCode [503]. Status [503 Service Unavailable]. Body: [baseType=error, code=ClusterUnavailable, message=ClusterUnavailable 503: cluster not found] from [https://rancher.example.org/v3/clusters/c-lrxgg?action=generateKubeconfig]
│ 
│   with rancher2_cluster.tf-test,
│   on main.tf line 12, in resource "rancher2_cluster" "tf-test":
│   12: resource "rancher2_cluster" "tf-test" {
│ 
╵
```

When provisioning a cluster without an RKE template, it is successful.

Version details:
```everholt:childCluster mgh$ ../bin/terraform version
Terraform v1.0.2
on darwin_amd64
+ provider registry.terraform.io/rancher/rancher2 v1.17.2

Your version of Terraform is out of date! The latest version
is 1.0.6. You can update by downloading from https://www.terraform.io/downloads.html
```

**Comment by rawmind0** [comment]
Hi @mgherman , same issue has already been reported. It should be fixed with PR https://github.com/rancher/terraform-provider-rancher2/pull/737

**Comment by rawmind0** [comment]
PR #737 alredy merged. the fix will be available at next tfp release v1.20.0
